src/main/java/com/wanderersoftherift/wotr/core/rift/RiftLevelManager.java

```diff
@@ -322,11 +325,21 @@ private static ServerLevel createRift(
         }
 
         int seed = config.seed().orElseGet(() -> new Random().nextInt());
+        LevelSettings levelSettings = worldData.getLevelSettings();
+        LevelSettings riftLevelSettings = new LevelSettings(
+                "rift", levelSettings.gameType(), levelSettings.hardcore(), levelSettings.difficulty(),
+                levelSettings.allowCommands(), levelSettings.gameRules(), levelSettings.getDataConfiguration(),
```

Comment:
Would it be possible to customise the game rule for the level at this point by creating a new GameRules, copying settings across and then setting fire tick?

Reply:
I guess the mixin might be the only way if you need to tweak the rule when loading the level too